src/sql/base/browser/ui/radioButton/radioButton.ts

```diff
@@ -33,7 +35,18 @@ export class RadioButton extends Widget {
 		this.label = opts.label;
 		this.enabled = opts.enabled || true;
 		this.checked = opts.checked || false;
-		this.onclick(this.inputElement, () => this._onClicked.fire());
+		this.onclick(this.inputElement, () => {
+			this._onClicked.fire();
+			let buttonGroup = document.getElementsByName(this.name);
+			buttonGroup.forEach((button) => {
+				let event = document.createEvent('HTMLEvents');
```

Comment:
Any reason you're creating an event here versus just firing the emitters manually? None of our other code seems to use this manual event creation and I don't know if there's any weird issues that might pop up because of this.

Reply:
I am going the HTMLEvents route because I do not have reference to other buttons. 
How can I get reference to an emitter that belong to other radio buttons?

Reply:
Yeah that's a good point. Hmmm. I had put some thought into this a while back but never came up with a solution I liked so dropped it. 

Since I can't really think of a better solution then sending the event seems reasonable enough for now. Make sure that we only handle this if the value is actually changed when doing stuff like updating the ext host though.

Also something to think about - but the groupings can be scoped. I don't know all the rules that govern that but for example if you have two separate forms on a page with radio buttons that all have the same name property they'll still each only be grouped with the other buttons in the form. Since we don't use forms AFAIK I'm hoping this isn't an issue (and generally people have been using unique names) but something to be aware of regardless.

Reply:
Yeah, that is one of the reasons I do not like the part 'getElementsByName'. This might generate an event for all elements that have the same 'name' attribute. Maybe I can introduce a custom attribute for radio buttons and set it to name and try to get elements by that custom attribute.  What do you think about that solution?

Reply:
Something to consider if we run into this issue - but for now I think this should be good enough. I don't see this being a common enough scenario to warrant extra custom logic like that.